tests/unit/getters/get-tx-amount-total.spec.js

```diff
@@ -52,31 +53,28 @@ const tests = [{
 })),
 ];
 
-const store = {
-  fungibleTokens: {
-    availableTokens: {
-      [STUB_TOKEN_CONTRACT_ADDRESS]: {
-        decimals: TEST_TOKEN_DECIMALS,
-      },
-    },
-  },
-};
-
 describe('getTxAmountTotal', () => {
+  const store = new Vuex.Store({});
+  const { availableTokens, getTxAmountTotal } = useFungibleTokens({ store });
+
+  availableTokens.value[STUB_TOKEN_CONTRACT_ADDRESS] = {
+    decimals: TEST_TOKEN_DECIMALS,
+  };
+
   tests.forEach((test) => it(
     `should return correct result for each type of transaction: for \
-${test.transaction.tx.type}/${test.transaction.tx.function}`,
+          ${test.transaction.tx.type}/${test.transaction.tx.function}`,
```

Comment:
Name of the test before this changes:
`should return correct result for each type of transaction: for PayingForTx/undefined`
name of the test after this changes:
`should return correct result for each type of transaction: for           PayingForTx/undefined`

Reply:
OK< I removed the white space.